def/flow.js

```diff
@@ -164,13 +168,23 @@ def("Identifier")
 def("TypeParameterDeclaration")
   .bases("Node")
   .build("params")
-  .field("params", [def("Identifier")]);
+  .field("params", [or(def("Identifier"), def("TypeParameter"))]);
 
 def("TypeParameterInstantiation")
   .bases("Node")
   .build("params")
   .field("params", [def("Type")]);
 
+def("TypeParameter")
+  .bases("Identifier")
```

Comment:
Yeah, I think so. `"Identifier"` from core would be fine, but since flow augments it with `typeParameter`—which `TypeParameter` doesn't have—I think you need to base it on `Type` and add a `"name"` field.

That said, this is the first time I've looked at this repo, so there might be a better way.